### DISCUSSION TITLE
Add a Benchmark

### DISCUSSION THREAD

**Discussion by LexiconCode** [DrewThomasson/ebook2audiobook#348] [General]
Thank you for making e-books accessible!

It will be great to have a standard benchmark that could run across all machines using a standard passage or short story `demo_mini_story_chapters_Drew.epub`. A text file could be generated programmatically at the end contain relevant information, system specs, processing time and ebook2audiobook version. People could share their benchmarks under discussions or you could even automated to basic web server to be displayed live in the documentation as a table.

**Comment by ROBERT-MCDOWELL** [comment]
this can be a feature indeed, we keep the idea on hold, more priorities are on rails now.

**Comment by ROBERT-MCDOWELL** [comment]
too much work for not enough user amount interest. closing.
